pkg/resource/v1/certconfig/create_test.go

```diff
@@ -8,10 +8,144 @@ import (
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
 	"github.com/giantswarm/certs"
+	"github.com/giantswarm/cluster-operator/pkg/resource/v1/certconfig/key"
+	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	clientgofake "k8s.io/client-go/kubernetes/fake"
+	k8stesting "k8s.io/client-go/testing"
 )
 
+func Test_ApplyCreateChange_Creates_createChange(t *testing.T) {
```

Comment:
I admit to be slightly lazy here and create separate test functions to test three basic scenarios instead of using table-driven approach. Reason for this is that I couldn't quickly come up with a good abstraction to implement understandable test case table and verify that correct set of `CertConfigs` are created. I think these three test cases cover most of the cases here, but just give me a note if you want me to try harder here.